### ISSUE TITLE
Laplace transformation of an ODE : not correctly displayed ?

### ISSUE THREAD

**Issue by paulvanliedekerke** [sympy/sympy#25815]
Following code performing a Laplace transformation on an ODE does not do what is expected to do, ie the constants with first/second derivatives are not appearing :
```
c_1, c_2 = sympy.symbols('c_1, c_2,',real=True, positive=True)
s = sympy.Symbol('s')
t = sympy.Symbol('t')
C = sympy.Function('C')
ode =  2*c_1*C(t).diff(t,2) + c_1*C(t).diff(t) +c_2*C(t)
sympy.Eq(ode,0)

L_ode = sympy.laplace_transform(ode, t, s)
L_ode
```
![Capture5](https://github.com/sympy/sympy/assets/32935549/29bbd434-869f-4da8-a260-c78ba9d46497)

**Comment by oscarbenjamin** [comment]
I get a different result with the pretty printer:
```python
In [6]: L_ode[0]
Out[6]: 
                                                  ⎛ 2                                         ⎛d       ⎞│   ⎞                                  
c₁⋅(s⋅LaplaceTransform(C(t), t, s) - C(0)) + 2⋅c₁⋅⎜s ⋅LaplaceTransform(C(t), t, s) - s⋅C(0) - ⎜──(C(t))⎟│   ⎟ + c₂⋅LaplaceTransform(C(t), t, s)
                                                  ⎝                                           ⎝dt      ⎠│t=0⎠  
```
LaTeX output with master:
```
In [8]: print(latex(L_ode[0]))
c_{1} \left(s \mathcal{L}_{t}\left[C{\left(t \right)}\right]\left(s\right) - C{\left(0 \right)}\right) + 2 c_{1} \left(s^{2} \mathcal{L}_{t}\left[C{\left(t \right)}\right]\left(s\right) - s C{\left(0 \right)} - \left. \frac{d}{d t} C{\left(t \right)} \right|_{\substack{ t=0 }}\right) + c_{2} \mathcal{L}_{t}\left[C{\left(t \right)}\right]\left(s\right)
```
What version of SymPy are you using? Have you tested with master?

**Comment by paulvanliedekerke** [comment]
I am using 1.10.1 via anaconda and using jupyter notebooks. Thanks for checking

**Comment by oscarbenjamin** [comment]
Can you test with at least SymPy 1.12 which is the latest release?

**Comment by paulvanliedekerke** [comment]
I would assume that it works on 1.12. Problem is that I cannot update to 1.12; 
conda update sympy gives:


Solving environment: \

Updating sympy is constricted by

anaconda -> requires sympy==1.10.1=py39haa95532_0

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`

done

==> WARNING: A newer version of conda exists. <==
  current version: 23.3.1
  latest version: 23.9.0

Please update conda by running

    $ conda update -n base -c defaults conda

Or to minimize the number of packages updated during conda update use

     conda install conda=23.9.0
# All requested packages already installed.

**Comment by oscarbenjamin** [comment]
The error message is telling you to update conda.

**Comment by paulvanliedekerke** [comment]
I know,  tried that multiple times but it didn't succeed (keeps on saying i am in version 23.3.1) , so guess I have to stay in that version.. or perhaps  you have another solution ?

**Comment by oscarbenjamin** [comment]
for the purposes of testing this out you don't need conda. You can just install Python from here:
https://www.python.org/downloads/
Then `pip install sympy` will give you the latest version of sympy (you might need to pip install other things like jupyter as well depending on how you use sympy).

**Comment by hanspi42** [comment]
Regarding your question, SymPy 1.11 onwards gives the correct solution; the Laplace transform was re-written heavily for 1.11.

Regarding anaconda, I think the best thing you could do is uninstall / re-install.

**Comment by hanspi42** [comment]
Since this is solved from 1.11 onwards, I close theb issue.
